### ISSUE TITLE
UI modifications

### ISSUE THREAD

**Issue by tmelliott** [iNZightVIT/iNZightModules#53]
- [ ] checkbox to fill symbols under Symbol
- [ ] toner-lite before toner 
- [ ] "Use Ranks" -> "Use Percentiles"
- [ ] Use Percentiles for opacify by (iNZightVIT/iNZightMaps)
- [ ] Overall opacity control as multiplier for opacify by

**Comment by daniel-barnett** [comment]
Percentiles for opacity is probably a bit more involved, but all the others have been pushed to [feature/mapfixes](https://github.com/iNZightVIT/iNZightModules/tree/feature/mapfixes). 

There's one last UI adjustment I'm making (clear buttons beside the variable dropdowns), then I can PR.

**Comment by tmelliott** [comment]
I can do the percentiles at some point, will just push updates as a minor patch.

**Comment by tmelliott** [comment]
Checking: @daniel-barnett did you ever do anything with percentiles, or were they forgotten about?

**Comment by daniel-barnett** [comment]
No, I haven't worked on the percentiles for opacity since this issue
